examples/tutorials/thrift/src/main/thrift/blog.thrift

```diff
@@ -0,0 +1,58 @@
+namespace java example.armeria.blog.thrift
+
+/**
+* A request object for creating a blog post!
+**/
+struct CreateBlogPostRequest {
+    1: string title;
+    2: string content;
+}
+
+struct GetBlogPostRequest {
+    1: i32 id;
+}
+
+struct ListBlogPostsRequest {
+    1: bool descending;
+}
+
+struct ListBlogPostsResponse {
+    1: list<BlogPost> blogs;
+}
+
+struct UpdateBlogPostRequest {
+    1: i32 id;
+    2: string title;
+    3: string content;
+}
+
+struct DeleteBlogPostRequest {
+    1: i32 id;
+}
+
+struct BlogPost {
+    1: i32 id;
+    2: string title;
+    3: string content;
+    4: i64 createdAt;
+    5: i64 modifiedAt;
+}
+
+exception BlogNotFoundException {
+    1: required string reason
```

Comment:
Could you share your rationale behind using `required`? 😄

Reply:
There was none 😅   removed